core/src/edu/asu/c3simulator/screens/CompanyPanel.java

```diff
@@ -0,0 +1,284 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.SelectBox;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup;
+import com.badlogic.gdx.scenes.scene2d.utils.Align;
+
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * This class displays the company - businesses tab in the council section of the game.
+ * 
+ * @author Alyahya, Mohammed
+ */
+public class CompanyPanel implements Screen
+{
+	@SuppressWarnings("unused")
+	private Game game;
+	private Stage stage;
+	private Skin skin;
+	
+	private SelectBox<String> businessSelectionBox;
+	private Label businessName, businessNetWorth, businessSellingPrice;
+	
+	public CompanyPanel(Game game)
+	{
+		this.game = game;
+		this.stage = new Stage();
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		
+		Table mainTable = new Table();
+		mainTable.setSize(0.70f * stage.getWidth(), 0.70f * stage.getHeight());
+		
+		Table upperSection = new Table();
+		Table lowerSection = new Table();
+		Actor businessSelection = createBusinessSelection();
+		Actor businessInfo = createBusinessInfo();
+		Actor businessGraph = createBusinessGraph();
+		Actor businessButtons = createBusinessButtons();
+		
+		upperSection.add(businessSelection).spaceRight(100).center();
+		upperSection.add(businessInfo).expand().fillX().left();
+		lowerSection.add(businessGraph).expand().center();
+		lowerSection.add(businessButtons).center().width(0.2f * mainTable.getWidth());
+		mainTable.add(upperSection).fillX();
+		mainTable.row();
+		mainTable.add(lowerSection).expand().fill();
+		
+		mainTable.setPosition(stage.getWidth() / 2 - mainTable.getWidth() / 2,
+				stage.getHeight() / 2 - mainTable.getHeight() / 2);
+		// upperSection.debug();
+		// lowerSection.debug();
+		// mainTable.debugTable();
```

Comment:
These lines should be removed. If you want to keep them for convenience, move them to a private method, and add an @unused annotation